inflamously/stable-cheese
# Develop to Master update

* Add proper requirements.txt
* Add simple pytests
* Add duplicate caption check
* Add logs text file support and fix some issues
